### DISCUSSION TITLE
Do we want to have enumerated list of common security exemptions?

### DISCUSSION THREAD

**Discussion by JSv4** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#147] [Suggestions] [answered]
**Per Issue #56, importing comments / suggestions to our static Google Doc into GitHub for discussion**

From @ashawber, here's a proposal to enumerate commonly-used exemptions. Right now the SecurityExemption type only has an unstructured `reason` field. I think Andy's proposing having an enumeration of the most common ones with possible an `other` choice and a text field for situations that don't fall under common exemptions. 

I think this is a good idea generally as it improves the quality of our structured data, but, as discussed in our TWG meetings earlier, you run into the issue of who maintains this list and who gets to decide what the commonly used exemptions are. This will change from time to time and jurisdiction to jurisdiction too, so you need someone maintaining it. TWG is probably not a great group for this, as this is more legal domain knowledge.

**Comment by JSv4** [comment, marked as answer]
I think this would be a good topic for discussion by the proposed standards group. We need a list of commonly-accepted / used exemptions and what to call them. Some are obv (Rule 506(b), Rule 144, etc.), but not all will be.

**Comment by ashawber** [comment]
Yes, there's a short list of the most common federal exemptions:

1. Rule 701
2. Rule 506(b)
3. Section 4(a)(2)
4. Rule 144
5. Section 4(a)(7)
